src/appengine/handlers/corpora.py

```diff
@@ -25,9 +25,9 @@
 class Handler(base_handler.Handler):
   """Manage data bundles."""
 
+  @handler.unsupported_on_local_server
```

Comment:
any reason for the order changes in this PR?

Reply:
It only mattered for

```
class LogoutHandler(base_handler.Handler):
  """Log out handler."""	  """Log out handler."""


   @handler.unsupported_on_local_server
  @handler.require_csrf_token	  @handler.require_csrf_token
  @handler.get(handler.HTML)	  @handler.get(handler.HTML)
```
where wrong order would mean we show the csrf error first. so i thought to fix the others as well. it is just unnecessary work, when instead we should be bailing out early.